pkg/lvmutils/utils.go

```diff
@@ -149,9 +149,39 @@ func UnmountLV(mountdir string) error {
 	return syscall.Unmount(mountdir, syscall.MNT_FORCE)
 }
 
+// IsDependentLvsError returns true if the error related to dependent Lvs exists
+func IsDependentLvsError(err error) bool {
+	if err == nil {
+		return false
+	}
+	return strings.Contains(err.Error(), "dependent volume(s). Proceed? [y/n]")
```

Comment:
A slight concern comparing strings here would be that string could be different for non-english locales.

Reply:
Tested this. Tried setting LANG to fr_FR.UTF-8 (french) and es_ES.UTF-8 (spanish). The output was always english. The lvm2 tools only have english strings. So a different locale should not affect this.

Reply:
Makes sense, unfortunately lvremove command is not providing different error codes. Worst case can happen if we fail to catch this failure is failing Delete call itself with respective error printed.